README.md

```diff
@@ -1,26 +1,43 @@
 # Backend Challenge
+
 ## Introduction
-Fat Llama relies heavily on our search in order for users to be able to find the items they need. The main two factors in the search are:
-- **Text match**: the user types a word or phrase that they want to find, and the search returns items that match this.
-- **Location**: the user indicates their location (through geolocation or through typing in the location search box) and the search returns items near the user
 
-On the production web & mobile app there are other factors that come into play such as lender rating, response time, categories, time since listing, and more. For this challenge though, we want you to focus only on the two main factors given above.
+Fat Llama relies heavily on our search in order for users to be able to find the items they need from other users. Our search must return a list of items matching a given search term combined with each item owners information.
+
+On the production web & mobile app there are other factors that come into play such as lender rating, response time, categories, time since listing, and more. For this challenge though, we want you to focus only on the above.
 
 ## The Challenge
-We want you to build a `GET /search` endpoint that will return the most appropriate 20 items given `searchTerm`, `lat` (latitude) and `lng` (longitude). e.g. `/search?searchTerm=camera&lat=51.948&lng=0.172943`. It is up to you to decide how to weight the two factors to return the most relevant results. We have provided you with a sqlite database containing just under 2000 items with the relevant fields.
+
+We want you to build a `GET /search` endpoint that will return the most appropriate 20 items given a `searchTerm`, e.g. `/search?searchTerm=camera`. The items returned must contain the owners information so we can correctly display them in the Fat Llama frontend. You are not required to incorporate any complex fuzzy text matching.
+
+The schema for the items returned should be:
+
+```javascript
+{
+  itemId: number,
+  itemName: string,
+  itemPrice: number,
+  userFirstName: string,
+  userCity: string
+}
+```
```

Comment:
I think maybe we should just have a picture of an item card with this info on, and leave them to come up with the schema. E.g. it might be nicer to structure it as:
```typescript
interface User {
  userFirstName: string
  userCity: string
}
interface ItemWithUser {
  itemId: number
  itemName: string
  itemPrice: number
  user: User
}
interface ApiResponse {
  page: number,
  results: ItemWithUser[]
}
```